fastapi_backend/pyproject.toml

```diff
@@ -1,32 +1,42 @@
-[tool.poetry]
+[project]
 name = "app"
 version = "0.1.0"
 description = ""
-authors = ["Anderson Resende <anderson@vinta.com.br>"]
+authors = [{ name = "Anderson Resende", email = "anderson@vinta.com.br" }]
+requires-python = ">=3.12,<3.13"
 readme = "README.md"
-package-mode = false
-packages = [{include = "commands"}]
+dependencies = [
+    "fastapi[standard]>=0.115.0,<0.116",
+    "asyncpg>=0.29.0,<0.30",
+    "fastapi-users[sqlalchemy]>=13.0.0,<14",
+    "pydantic-settings>=2.5.2,<3",
+    "fastapi-mail>=1.4.1,<2",
+    "pytest-asyncio>=0.24.0,<0.25",
+    "migrate-to-uv>=0.2.1,<0.3",
```

Comment:
I think you can remove migrate-to-uv now?

Reply:
thanks